index.d.ts

```diff
@@ -0,0 +1,293 @@
+/// <reference types="request" />
+/// <reference types="node" />
+
+import * as request from 'request';
```

Comment:
From `request` I use only `request.CoreOptions` and `request.RequestResponse`: [here](https://github.com/jira-node/node-jira-client/pull/194/files#diff-b52768974e6bc0faccb7d4b75b162c99R180)

And these types are the same for `request` and `request-promise` packages